query/config_cache.go

```diff
@@ -156,3 +159,84 @@ func FindConfigForComponent(ctx context.Context, componentID, configType string)
 	err := query.Find(&dbConfigObjects).Error
 	return dbConfigObjects, err
 }
+
+// <id>/<related_type> -> []related_ids
+var configItemRelatedTypeCache = cache.New[[]string](gocache_store.NewGoCache(gocache.New(6*time.Hour, 6*time.Hour)))
+
+func configItemRelatedTypeCacheKey(id, typ string) string {
+	return "configRelatedType:" + id + typ
+}
+
+// <id> -> models.ConfigItem
+var configItemCache = cache.New[models.ConfigItem](gocache_store.NewGoCache(gocache.New(6*time.Hour, 6*time.Hour)))
+
+func configItemCacheKey(id string) string {
+	return "configID:" + id
+}
+
+// <config_id> -> []related_ids
+var configRelationCache = cache.New[[]string](gocache_store.NewGoCache(gocache.New(6*time.Hour, 6*time.Hour)))
+
+func configRelationCacheKey(id string) string {
+	return "configRelatedIDs:" + id
+}
+
```

Comment:
10 mins for cache time, job runs every 5 mins

Reply:
@yashmehrotra is there a commit missing ?

Reply:
Sorry, forgot to push the commit